src/frontend/utils-frontend.ts

```diff
@@ -26,14 +26,50 @@ export const OverrideTurretInputHeightForReactSelectDiv = styled.div`
   input {
     height: auto;
   }
-`
+`;
 
 export const StyledDataTableWrapperDiv = styled.div`
-  .rdt_TableCell :focus, .rdt_TableCol_Sortable:focus {
+  .rdt_TableCell :focus,
+  .rdt_TableCol_Sortable:focus {
     outline: 0.125rem solid #08c;
     outline-offset: -0.125rem; // this overrides the normal offset to avoid
-                               // drawing the outline outside of the parent
-                               // which would cause it to disappear because of
-                               // overflow: hidden; definitions.
+    // drawing the outline outside of the parent
+    // which would cause it to disappear because of
+    // overflow: hidden; definitions.
   }
-`
\ No newline at end of file
+`;
+
+export const StyledNav = styled.nav`
```

Comment:
Because navigation will be only ever used in one place, I think it is better to just put this to `Navigation.tsx`